code/headers/display_adapter.hpp

```diff
@@ -1,8 +1,21 @@
 #pragma once
 
 #include <hwlib.hpp>
-
+#include <display_cursor.hpp>
 namespace r2d2::display {
+    /*
+    * Class display_c is the base class for all displays in R2D2. It inherits
+    * from hwlib::window.
+    * 
+    * @tparam Cursor_Count is the amount of cursors to store. Most of the time 
+    * this is equal to the number claimed cursors in the display_cursor enum.
+    * 
+    * @tparam Display_Size_Width is an uint8_t representing the width of the display
+    * 
+    * @tparam Display_Size_Height is an uint8_t representing the height of the display
+    * 
+    */
+    template<std::size_t Cursor_Count, uint8_t Display_Size_Width, uint8_t Display_Size_Height>
```

Comment:
Template parameters should be PascalCase. As in the styleguide:

https://github.com/R2D2-2019/R2D2-2019/wiki/Style-Guide#naming-and-capitals

Reply:
Fixed in c2dfd0d4e42b03c5f9eeb9234d5f7f92fd062ad3